src/controllers/dlgprefcontroller.cpp

```diff
@@ -21,6 +21,17 @@
 #include "preferences/usersettings.h"
 #include "util/version.h"
 
+QString nameForPreset(const PresetInfo& preset);
+namespace {
+    bool presetInfoNameComparator(const PresetInfo &a, const PresetInfo &b) {
```

Comment:
May be I have to read the coding guideline I don't know about them, my coding style is the result of reading style and applying from the book